# control/arm_control/config/arm_control_sim.yaml
controller_manager:
  ros__parameters:
    update_rate: 100

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    arm_controller:
      type: joint_trajectory_controller/JointTrajectoryController
      joints:
        - base_rotation_joint
        - shoulder_pitch_joint
        - elbow_pitch_joint
        - wrist_pitch_joint
        - wrist_roll_joint
      command_interfaces:
        - position
      state_interfaces:
        - position
        - velocity

# Hardware simulation plugin
gazebo_ros2_control:
  ros__parameters:
    use_sim_time: true
